CMakeLists.txt

```diff
@@ -712,8 +712,8 @@ else()
 endif()
 
 # Boost
-find_public_dependency(Boost REQUIRED COMPONENTS system)
-target_include_directories(torrent-rasterbar PUBLIC ${Boost_INCLUDE_DIRS})
+find_package(Boost REQUIRED COMPONENTS system)
```

Comment:
`find_public_dependency` ensures that libtorrent clients will look for Boost too. Can't understand this change.

Reply:
I see. I thought `PUBLIC` did that. I will change this back. However, the fundamental problem of it not working right is still there..